index.html

```diff
@@ -138,26 +145,28 @@ <h3 class="black bold cap">Multi-Post Stories</h3>
 
   <!-- about -->
   <section class="about margin-seprator bg">
-    <div class="wrapper hero flex space-between">
-      <h2 class="heading bold cap margin-seprator">About <br> Myself</h2>
-      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-      <a class="bold upper" href="#contact-form">let's connect</a>
-      <ul class="social flex cross-center">
-        <li><a href="https://twitter.com/ramoun16"><img src="./images/social/twitter.svg" alt=""></a></li>
-        <li><a href="https://www.linkedin.com/in/ramoun"><img src="./images/social/linkedin.svg" alt=""></a></li>
-        <li><a href="https://medium.com/@omartarekramoun"><img src="./images/social/medium.svg" alt=""></a></li>
-        <li><a href="https://github.com/mrramoun"><img src="./images/social/github.svg" alt=""></a></li>
-        <li><a href="https://angel.co/u/omar-ramoun"><img src="./images/social/vector.svg" alt=""></a></li>
-      </ul>
-      <button class="bg round-borders cap" type="button">get my resume</button>
+    <div class="wrapper hero flex space-between flex-column">
+      <div class="left-side hero flex space-between flex-column">
+        <h2 class="heading bold cap margin-seprator">About <br> Myself</h2>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a class="bold upper" href="#contact-form">let's connect</a>
+        <ul class="social flex cross-center">
+          <li><a href="https://twitter.com/ramoun16"><img src="./images/social/twitter.svg" alt=""></a></li>
+          <li><a href="https://www.linkedin.com/in/ramoun"><img src="./images/social/linkedin.svg" alt=""></a></li>
+          <li><a href="https://medium.com/@omartarekramoun"><img src="./images/social/medium.svg" alt=""></a></li>
+          <li><a href="https://github.com/mrramoun"><img src="./images/social/github.svg" alt=""></a></li>
+          <li><a href="https://angel.co/u/omar-ramoun"><img src="./images/social/vector.svg" alt=""></a></li>
+        </ul>
+        <button class="bg round-borders cap" type="button">get my resume</button>
+      </div>
       <ul class="skills hero flex flex-column full-width">
         <li>
           <details open>
             <summary class="summary flex cross-center space-between cap">
               <span>languages</span>
               <span class="arrow flex"><img src="./images/arrow.svg" alt=""></span>
             </summary>
-            <ul class="hero">
+            <ul class="languages hero">
```

Comment:
- [ ] [OPTIONAL] You can improve your responsive design also for large screens, **the red line is the current width**, and **the blue line is the width expected**, you can use the `max-width CSS attribute in order to avoid this kind of issue.

![image](https://user-images.githubusercontent.com/60164239/130311049-3a3dd9df-fd0a-4f63-a349-c9eb9aba59aa.png)